pyseir/icu/utils.py

```diff
@@ -0,0 +1,33 @@
+from libs.datasets import combined_datasets
+from covidactnow.datapublic.common_fields import CommonFields
+import pandas as pd
+import us
+
+
+def _cases_quantile_range(x):
+    """Compute the quantile range for the cumulative case data.  We sometimes get single points of
+    a timeseries that are obviously wrong. Taking the quantile range is less sensitive to outliers
+    than either just "last minus first" or "maximum minus minimum"
+    """
+    QUANTILES = [0.05, 0.95]  # We sometimes get single point obviously wrong input.
+    # Quantile range is less sensitive than max-min or last-first.
+    lower, upper = x[CommonFields.CASES].quantile(q=QUANTILES).values
```

Comment:
I would rather this function take in a raw pandas series.  Makes it easier to test.

Reply:
Done.